### PR TITLE
Install SUSE certs and enable SSL warnings

### PR DESCRIPTION
**Pull request by ricardobranco777**
Install SUSE certs and enable SSL warnings

### CODE REVIEW

**Review by pdostal — DISMISSED** [review]

File: ocw/lib/openqa.py [hunk 1]
```diff
@@ -26,7 +18,7 @@ def get_url(server):
             got = openqa_client.client.requests.head(
                 url,
                 timeout=5,
-                verify=bool(os.environ.get('REQUESTS_CA_BUNDLE')),
+                verify=False,
```
Comment (dcermak) [inline ocw/lib/openqa.py:21]: You meant this, right?
```suggestion
                verify=True,
```

File: ocw/lib/openqa.py [hunk 2]
```diff
@@ -48,7 +40,7 @@ def __new__(cls, **kwargs):
     def __init__(self, **kwargs):
         kwargs.pop("server")
         self.__client = openqa_client.client.OpenQA_Client(server=self.server, **kwargs)
-        self.__client.session.verify = bool(os.environ.get('REQUESTS_CA_BUNDLE'))
+        self.__client.session.verify = False
```
Comment (clanig) [inline ocw/lib/openqa.py:43]: This is disabling TLS verification as well.

Comment (ricardobranco777) [reply ocw/lib/openqa.py:43]: > This is disabling TLS verification as well.

We can't verify every openQA instance with self-signed certs.

File: ocw/lib/openqa.py (hunk 1 above)
Comment (grisu48) [reply ocw/lib/openqa.py:21]: I was also puzzled by this but this is valid.

The reason is that `pcw` needs to query any openQA instances for job status updates. This includes openQA development instances, where many people use self-signed certificates.

**Review by grisu48 — APPROVED** [review]

File: ocw/lib/openqa.py (hunk 1 above)
Comment (clanig) [reply ocw/lib/openqa.py:21]: OK for me as well then. We might find a better overall solution to everything later.
However, I would appreciate to have a short comment why TLS verification is disabled whenever this is done in any code. @ricardobranco777 I would appreciate very much if you could add a short comment to the sections.

File: ocw/lib/openqa.py (hunk 2 above)
Comment (clanig) [reply ocw/lib/openqa.py:43]: OK, I see you are right. Thanks for the explanation!

**Comment by clanig** [comment]
Thank you for the changes. Looks great from my perspective.

File: Dockerfile [hunk 3]
```diff
@@ -28,5 +29,7 @@ VOLUME /pcw/db
 
 EXPOSE 8000/tcp
 
+ENV REQUESTS_CA_BUNDLE=/etc/ssl/ca-bundle.pem
```
Comment (grisu48) [inline Dockerfile:32]:
```suggestion
# Required to use system certs in python-requests
ENV REQUESTS_CA_BUNDLE=/etc/ssl/ca-bundle.pem
```

**Review by grisu48 — APPROVED** [review]

**Comment by asmorodskyi** [comment]
I think we also need to tweak Dockerfile_dev so we won't see this warnings during developing new features

**Pull request merged**